src/gui/GuiWindow.zig

```diff
@@ -367,7 +367,7 @@ pub fn updateHovered(self: *GuiWindow, mousePosition: Vec2f) void {
 }
 
 pub fn updateWindowPosition(self: *GuiWindow) void {
-	if(self.contentSize[0] < iconWidth*4) {
+	if(self.showTitleBar and self.contentSize[0] < iconWidth*4) {
```

Comment:
This is not fixing the issue, it's just hiding it. All windows have a titlebar when `reorderWindows` is set to true.

What I would like you to do instead is to look at the problematic windows and figure out why they are not wide enough.

Reply:
It seems I just didn't take into account the amount of icons. I just assumed there would be 3, but there are 2 when the window isn't closable. Both windows I see in the logs are 64px, and non-closable.

Reply:
I still get
```
Window chat width is too small: 16px before scaling
Resized width to 54px unscaled
```
when entering the world.

Reply:
I must not have noticed. I'll take a look at that window.

Reply:
It seems to be an issue with the dynamic sizing of that window. I refactored the minimum size check so that the dynamic sizing of that window can make sure to set the size of that window to at least the minimum itself.